configuring/configuring-build-runs.adoc

```diff
@@ -7,9 +7,13 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 
-In a `BuildRun` custom resource (CR), you can define the build reference, build specification, parameter values, service account, output, retention parameters, and volumes to configure a build run. A `BuildRun` resource is available for use within a namespace.
+In a `BuildRun` custom resource (CR), you can define the build reference, build specification,
```

Comment:
```suggestion
In a `BuildRun` custom resource (CR), you can configure a build run by defining the build reference, build specification, parameter values, service account, output, retention parameters, and volumes to configure a build run. You can also configure build pods by using a `Build` CR.
```
From the [OCP doc guidelines](https://92126--ocpdocs-pr.netlify.app/openshift-builds/latest/configuring/configuring-build-runs.html
https://92126--ocpdocs-pr.netlify.app/openshift-builds/latest/configuring/configuring-openshift-builds.html): "Do not hard wrap lines at 80 characters (or at any other length)."